acquamanCommon.pri

```diff
@@ -467,9 +467,9 @@ HEADERS += \
 	source/ui/beamline/AMSlitView.h  \
 	source/beamline/AMSlits.h \
 	source/ui/beamline/AMSlitsView.h \
-	source/beamline/AMTemperatureMonitor.h \
-	source/beamline/AMTemperatureMonitorGroup.h \
-	source/beamline/AMTemperatureMonitorGroupStatus.h
+    $$PWD/source/beamline/AMBeamlineControlGroupStatus.h \
+    $$PWD/source/beamline/AMBeamlineControlGroup.h \
+    $$PWD/source/beamline/AMBeamlineControl.h
```

Comment:
I imagine there must be a setting for this somewhere. I can't see anything in Creator preferences yet, and I'm having trouble finding something relevant on Google too.